lib/SILGen/SILGenPattern.cpp

```diff
@@ -2170,23 +2209,22 @@ void PatternMatchEmission::emitEnumElementDispatch(
       if (eltTL->isLoadable()) {
         // If we do not have a loadable value, just use getManagedSubObject
```

Comment:
Nitpick: It's getManagedSubobject() not getManagedSubObject()

Reply:
Going to do this in a follow on commit.